### ISSUE TITLE
Default GraphIEC setting causes timing issues

### ISSUE THREAD

**Issue by ypsvlq** [pi1541/Pi1541#245]
The options.txt in the repository and setup zip sets `GraphIEC = 1`, which has caused hard to diagnose issues for me on an RPi 3B and others (https://www.lemon64.com/forum/viewtopic.php?p=882574#882574) -- I spent a lot of time trying to find problems with C64, SD card, cable, etc. before finding that post.

As it's known to cause issues and doesn't seem like an essential feature, I think it be disabled by default.

**Comment by perhof** [comment]
I totally agree and I have suggested that to Steve earlier but he didn't seem to care.
I'm pretty sure that he doesn't read these tickets either.
